core/templates/dev/head/pages/exploration-editor-page/feedback-tab/services/thread-data.service.spec.ts

```diff
@@ -123,20 +104,14 @@ describe('retrieving threads service', function() {
       suggestion_thread_dicts: mockSuggestionThreads
     });
     $httpBackend.whenGET(
-      '/suggestionlisthandler?target_type=exploration&target_id=' + expId
-    ).respond({ suggestions: mockSuggestions });
-
+      '/suggestionlisthandler?target_type=exploration&target_id=' + expId)
+      .respond({ suggestions: mockSuggestions });
 
     ThreadDataService.fetchThreads().then(threadData => {
-      for (let feedbackThread of mockFeedbackThreads) {
-        expect(threadData.feedbackThreads).toContain(jasmine.objectContaining(
-          { threadId: feedbackThread.thread_id }));
-      }
-
-      for (let suggestionThread of mockSuggestionThreads) {
-        expect(threadData.suggestionThreads).toContain(jasmine.objectContaining(
-          { threadId: suggestionThread.thread_id }));
-      }
+      expect(threadData.feedbackThreads.map(t => t.threadId))
```

Comment:
I tried that first, but then I had to use `jasmine.objectContaining` to look for the thread id. I'm strongly against doing that because we should be working with the domain objects.

Having trouble thinking of a cleaner way to do it, since I don't really like it either :(